COMPONENT-OWNERS.md

```diff
@@ -0,0 +1,93 @@
+# CloudNativePG Component Owners
+
+## charts
+
+**Repository:** [cloudnative-pg/charts](https://github.com/cloudnative-pg/charts)
```

Comment:
I'd suggest splitting the two charts into two separate components 🙏

Reply:
separate components or separate repos? by components you mean like add a CODEOWNERS files in the top of the project and split them from there ?

Reply:
What about now?

Reply:
Nice! I think it looks better and more clear now, what do you think @phisco ?